src/freva/_history.py

```diff
@@ -95,14 +95,16 @@ def history(
             f"{prefix}, limit={limit}, since={since},"
             f" until={until}, entry_ids={entry_ids}"
         )
+
     rows = pm.get_history(
-        user=None,
+        user=None if user == "all" or user == "*" else User(user),
```

Comment:
Please avoid those inline if-else clauses. 

If you follow my suggestion from above in the `get_history` method you don't need to create a user obeject. You could simply pass the user name with the new user_name argument.

Reply:
done